app/models/chargeback.rb

```diff
@@ -1,4 +1,6 @@
 class Chargeback < ActsAsArModel
+  singleton_class.send(:prepend, DescendantLoader::ArDescendantsWithLoader)
```

Comment:
@jrafanie specs are failing with `DescendantLoader.instance.load_subclasses(self)` by this spec:
```
bundle exec rspec spec/models/chargeback_container_project_spec.rb (also for chargeback_vm_spec.rb)
```

```
`load_missing_constant': Circular dependency detected while autoloading constant ChargebackContainerProject (RuntimeError)
```

when I redefine method like it here in chargeback.rb, it works 
https://github.com/ManageIQ/manageiq/blob/master/lib/extensions/descendant_loader.rb#L251

so should I use as it was `singleton_class.send(:prepend, DescendantLoader::ArDescendantsWithLoader)` ?